### PR TITLE
Graphviz 2.44.1

### PR DESCRIPTION
**Pull request by ngerke**
Specify library name and version:  **graphviz/2.44.1**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/graphviz/2.44.1/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                packages = ["libtool-ltdl-devel"]
+            elif tools.os_info.with_pacman:
+                packages = ["libtool"]
+            elif tools.os_info.with_zypper:
+                packages = ["libltdl7"]
+            else:
+                self.output.warn("Do not know how to install 'libltdl' for {}.".format(tools.os_info.linux_distro))
+                packages = []
+            for p in packages:
+                package_tool.install(update=True, packages=p)
```
Comment (madebr) [inline recipes/graphviz/2.44.1/conanfile.py:68]: No need for installing system requirements.
All requirements of graphviz are available on cci.

File: recipes/graphviz/2.44.1/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            self.build_requires("winflexbison/2.5.22")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        folder_name = "%s-%s" % (self.name, self.version)
+        os.rename(folder_name, self._source_subfolder)
+        # Get windows dependencies from official repo
+        if tools.os_info.is_windows and self.settings.os == "Windows":
+            git = tools.Git(folder=os.path.join(self._source_subfolder, "windows", "dependencies", "libraries"))
+            git.clone("https://github.com/ErwinJanssen/graphviz-windows-dependencies.git", branch="141d3a21be904fa8dc2ae3ed01d36684db07a35d", shallow=True)
```
Comment (madebr) [inline recipes/graphviz/2.44.1/conanfile.py:42]: Please use cci for all dependencies.

File: recipes/graphviz/2.44.1/conanfile.py (hunk 1 above)
Comment (ngerke) [reply recipes/graphviz/2.44.1/conanfile.py]: Oh I didn't saw the libtool pkg. I Updated it.

File: recipes/graphviz/2.44.1/conanfile.py (hunk 2 above)
Comment (ngerke) [reply recipes/graphviz/2.44.1/conanfile.py]: I was able to get all dependencies except for rxspencer. I had a quick try on building it myself, but unless I use the provided lib I get compile errors. I need to have a deeper look at this.

**Comment by ericriff** [comment]
Can we give this a second chance?

**Pull request closed without merging**